mir-module/src/main/resources/xsl/metadata/static/mir-admindata-box-static.xsl

```diff
@@ -129,4 +133,30 @@
       </div>
     </div>
   </xsl:template>
-</xsl:stylesheet>
\ No newline at end of file
+
+  <xsl:template name="print-user-info">
+    <xsl:param name="user"/>
+    <xsl:param name="label"/>
+
+    <tr>
+      <td class="metaname">
+        <xsl:value-of select="concat($label, ':')"/>
+      </td>
+      <td class="metavalue">
+        <xsl:choose>
+          <xsl:when test="$user/user/realName">
+            <xsl:attribute name="title">
+              <xsl:value-of select="$user/user/@name"/>
+            </xsl:attribute>
+            <xsl:value-of select="$user/user/realName"/>
+          </xsl:when>
+          <xsl:otherwise>
+            <xsl:value-of select="$user/user/@name"/>
+          </xsl:otherwise>
+        </xsl:choose>
+      </td>
+    </tr>
+
```

Comment:
Why reinvent the layout rendering instead of deferring to the existing `printMetaDate` template?

Suggestion:

```
  <xsl:template name="print-user-info">
    <xsl:param name="user"/>
    <xsl:param name="label"/>

    <xsl:variable name="realName">
      <xsl:choose>
        <xsl:when test="$user/user/realName">
          <xsl:value-of select="concat($user/user/realName,' (',$user/user/@name,')')"/>
        </xsl:when>
        <xsl:otherwise>
          <xsl:value-of select="$user/user/@name"/>
        </xsl:otherwise>
      </xsl:choose>
    </xsl:variable>

    <xsl:call-template name="printMetaDate">
      <xsl:with-param select="exslt:node-set($realName)" name="nodes"/>
      <xsl:with-param select="$label" name="label"/>
    </xsl:call-template>

  </xsl:template>

Reply:
But then no tooltip will be displayed.

Reply:
What tooltip? Using the stylesheet from your PR, the following HTML was generated:

```
<dl>
<dt>Publikationsstatus:</dt>
<dd>angelegt</dd>
<dt>Erstellt am:</dt>
<dd>25.07.2024 - 07:54:12</dd>
<dt>Erstellt von:</dt>
<dd>Superuser</dd>
<dt>Letzte Änderung:</dt>
<dd>25.07.2024 - 08:38:04</dd>
<dt>Geändert von:</dt>
<dd>Superuser</dd>
<dt>MyCoRe ID:</dt>
<dd>macau_mods_00000001</dd>
<dt>Version:</dt>
<dd>20<br>
<a style="cursor: pointer" id="historyStarter">(Gesamte Versionsgeschichte)</a>
</dd>
</dl>
```

As I said, the `title`-attribute doesn't appear to survive the layouting process.

Reply:
This is strange since I added a modification in mods-metadata-page.xsl to retain the title attribute

Reply:
Even then, I'd like my approach better, because it is impossible (or at least cumbersome) to copy the username from a tooltip.

Copying the username i a usual step in many administrative tasks, especially when trying to investigate some issues.

Reply:
> Specifically, [this line](https://github.com/MyCoRe-Org/mir/blob/622b47518e4ab8780a9aee783b2cf4e89c7ae570/mir-module/src/main/resources/xslt/metadata/mods-metadata-page.xsl#L231) in `mods-metadata-page.xsl` would have to be changed from `<xsl:copy-of select="node()|*"/>` to `<xsl:copy-of select="node()|*|@*"/>` for the attribute to survive.

this would copy all attributes `@class` included and might infer with existing layouts. I do get your point with copying the username to the clipboard. Maybe linking to the user profile page would be an option?

Reply:
Okay, maybe I didn't integrate your change to `mods-metadata-page.xsl` properly into my test setup (I couldn't just apply the PR because I'm in the middle of something else right now), so please disregard my comments about the attribute not surviving the layout process. My bad. Sorry for the inconvenience.

Reply:
Linking could indeed be fine. I'm not sure right now, is the admin metadata box actually only ever displayed to admins, i.e. to users who would actually be allowed to see the linked user profile page?

Reply:
Usually only admins can see the box but in my repo the creators are allowed to see them as well. Unfortunately the user cannot be determined during display since the content is static.

Reply:
If it can not be guaranteed the the admin metadata box will only be displayed to admins, linking to the user profile might not be the best idea.